rust/worker/src/blockstore/arrow/block/types.rs

```diff
@@ -92,6 +92,118 @@ impl Block {
         None
     }
 
+    pub fn get_prefix<'me, K: ArrowReadableKey<'me>, V: ArrowReadableValue<'me>>(
+        &'me self,
+        prefix: &str,
+    ) -> Option<Vec<(&str, K, V)>> {
```

Comment:
nit: We can just return a `Vec<(K, V)>` right? The `&str` will always equal `prefix`

Reply:
Same comment for below but I won't repeat it

Reply:
I was honoring the top level blockfile API - `get_by_prefix`. Seems like that can be changed too